shells/electron/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "main": "./index.js",
+  "scripts": {
+    "build": "webpack --config webpack.backend.js && webpack --config webpack.config.js"
+  },
+  "dependencies": {
+    "ws": "^0.7.2"
```

Comment:
why 0.7.2? 1.1.0 is latest.

0.7.2 seems like this requires an older version of node. Should that be added to the engines field so consumers know of that requirement?